runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -503,6 +496,17 @@ private void removeCounter()
 		counter = null;
 	}
 
+	private void updateCounter()
+	{
+		if (!config.showInfobox())
+		{
+			return;
+		}
+
+		addCounter();
+		counter.setText(String.valueOf(amount));
```

Comment:
Make sure to add `infoTimer = Instant.now();`

Reply:
I'm pretty sure I added it when I made the changes first time, but I had to re-do them. Thanks for pointing that out.